src/lib-components/TwoColLayoutWStickySideBar.vue

```diff
@@ -111,10 +109,18 @@ onMounted(() => {
 @media (max-width: 1200px) {
   .two-column {
     padding-right: var(--unit-gutter);
-  }
 
-  .two-column>.primary-column {
-    width: 62%;
+    .primary-column {
+      width: 62%;
+
+      .primary-section-wrapper {
+        padding-left: var(--unit-gutter);
```

Comment:
are we sure we need the padding-left at the medium size as well as the small? Its hard to tell until we can see the content on the page.